src/components/CardList.jsx

```diff
@@ -1,47 +1,47 @@
 import styled from 'styled-components';
+import { useEffect, useState } from "react";
 
 function CardList() {
+
+  const apiURL="https://rickandmortyapi.com/api/character";
+  const [characters, setCharacters] = useState([]);
+
+  const Characters =() =>{
+    fetch(apiURL).then((response) => response.json()).then((data) => setCharacters(data.results))
+  }
+
+
+  useEffect(() => {
+    Characters();
+  }, []);
```

Comment:
Thomas gave the tip today to write the `fetch()` function in the `App.jsx`. Will be necessary later.
So I would recommend you to do that already now.

Reply:
Thank you very much Alex. I will do it in my next step. 👍